alphacsc/update_d_multi.py

```diff
@@ -215,6 +242,8 @@ def grad_u(u):
             if adaptive_step_size:
                 Lu = 1
             else:
+                if loss != 'l2':
+                    raise NotImplementedError()
```

Comment:
Haven't all this code been moved in `_solver_d.py`? I would havoid having duplicated code here if this is not to be called anymore.